shared/components/Footer.tsx

```diff
@@ -1,14 +1,209 @@
-import { useTranslation } from 'next-i18next';
-import { IS_IN_MAINTENANCE } from '../utils/constants';
+import {
+  Divider,
+  Flex,
+  HStack,
+  Link,
+  Text,
+  Stack,
+  VStack,
+  Icon,
+  Square,
+} from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
+import { IS_IN_MAINTENANCE } from "../utils/constants";
+import FacebookIcon from "./Icons/FacebookIcon";
+import InstagramIcon from "./Icons/InstagramIcon";
+import YoutubeIcon from "./Icons/YoutubeIcon";
+
+import Logo from "./Logo";
 
 export default function Footer() {
-  const { t } = useTranslation('footer');
+  const { t } = useTranslation("footer");
 
   if (IS_IN_MAINTENANCE) {
-    return null
+    return null;
   }
 
   return (
-    <footer>{t("title")}</footer>
-  )
-}
\ No newline at end of file
+    <Flex
+      width={"100%"}
+      bg={"black"}
+      paddingTop={"75px"}
```

Comment:
bora usar o spacing do chakra, esse site ajuda a converter: https://nekocalc.com/px-to-rem-converter